csharp/src/Ice/Object.cs

```diff
@@ -156,8 +152,19 @@ public ValueTask<OutgoingResponse> dispatchAsync(IncomingRequest request)
     }
 }
 
+/// <summary>
+/// Base class for asynchronous dynamic dispatch servants.
+/// </summary>
+/// <remarks>This class is provided for backward compatibility. You should consider deriving directly from
+/// <see cref="Object"/> and overriding the <see cref="Object.dispatchAsync"/> method.</remarks>
 public abstract class BlobjectAsync : Object
 {
+    /// <summary>
+    /// Dispatches an incoming request.
+    /// </summary>
+    /// <param name="inEncaps">An encapsulation containing the encoded in-parameters for the operation.</param>
+    /// <param name="current">The Current object of the incoming request.</param>
+    /// <returns>A task that will complete an instance of <see cref="Object_Ice_invokeResult"/>.</returns>
```

Comment:
this sounds odd, should it be `complete with an instance ...`?

Reply:
Yes, fixed!